projects.json

```diff
@@ -117,7 +117,7 @@
       "Friends and global leaderboard!."
     ],
     "footer": [
-      "<a href='https://play.google.com/store/apps/details?id=org.mdou.nienie'>Google Play</a>"
+      "<a href='https://apkpure.com/nie-nie/org.mdou.nienie/download?from=details'>Download here</a>"
```

Comment:
not sure we should use this kind of apk mirrors link here, not sure why the app was removed from play store, probably better to contact author about it.